vscode/src/rubyLsp.ts

```diff
@@ -287,24 +287,32 @@ export class RubyLsp {
         }
 
         const options: vscode.QuickPickItem[] = client.addons
-          .sort((addon) => {
-            // Display errored addons last
-            if (addon.errored) {
-              return 1;
-            }
-
-            return -1;
-          })
+          .sort((addon) => (addon.errored ? 1 : -1))
           .map((addon) => {
             const icon = addon.errored ? "$(error)" : "$(pass)";
             return {
               label: `${icon} ${addon.name} ${addon.version ? `v${addon.version}` : ""}`,
             };
           });
 
-        await vscode.window.showQuickPick(options, {
-          placeHolder: "Addons (readonly)",
+        const quickPick = vscode.window.createQuickPick();
+        quickPick.items = options;
+        quickPick.placeholder = "Addons (click to view output)";
+
+        quickPick.onDidAccept(() => {
+          const selected = quickPick.selectedItems[0];
+          // Ideally, we should display information that's specific to the selected addon
```

Comment:
If we provide an API in the server that tags logs with the add-on name, then you might be able to reveal the output channel with a filter applied, so that you only see the output related to that add-on.

Reply:
Yeah that will be great too 👍